README.md

```diff
@@ -82,12 +90,35 @@ Create an application from a template:
 cartridge create --name myapp
 ```
 
+Build an application:
+
+```sh
+cartridge build ./myapp
+```
+
+Run instances locally:
+
+```sh
+cartridge start
+cartridge stop
+```
+
 Pack an application into a distributable, for example into an RPM package:
 
 ```sh
 cartridge pack rpm ./myapp
 ```
 
+### Building an application
+
+You can call `cartridge build` command to build application locally.
+It can be useful for local development.
+
+These steps will be performed on running this command:
+
+* running `cartridge.pre-build` or `.cartridge.pre` [DEPRECATED];
```

Comment:
It could confuse users. It's not clear `.cartridge.pre` is deprecated or both '`cartridge.pre-build` or `.cartridge.pre`'

Reply:
I agree with you.

Maybe

running `cartridge.pre-build` (or  [DEPRECATED] `.cartridge.pre`);

Reply:
Yes, it's better